dt-reports/magic-url-endpoints.php

```diff
@@ -121,7 +132,24 @@ public function email_magic( WP_REST_Request $request ) {
             }
 
             // check if email exists to send to
-            if ( ! isset( $post_record['contact_email'][0] ) && empty( $params['email'] ) ) {
+            $emails = [];
+            if ( isset( $params['email'] ) && ! empty( $params['email'] ) ){
+                $emails[]   = $params['email'];
+            }
+            else if ( isset( $post_record['contact_email'][0] ) ) {
+                $emails[]  = $post_record['contact_email'][0]['value'];
+            }
+            else if ( isset( $post_record[$target_field] ) && ! empty( $post_record[$target_field] ) ) {
```

Comment:
Else if the target connection field (default 'reporter') exists, then loop through and lookup available emails found in these contacts and send links to these individuals.